### ISSUE TITLE
New watch option does not report errors

### ISSUE THREAD

**Issue by montogeek** [rollup/rollup#712]
Hi!

When using the new watch flag (`-w`) it does not report bundling errors, such as `Unexpected token` and similars.

When running without the `-w` flag it does report errors.

Example:
```
montogeek ~/foo/bar [03:27 PM]→ 😛  (dev)
👉  rollup -c rollup.config.umd.js -w
checking rollup-watch version...
bundling...
montogeek ~/foo/bar [03:31 PM]→ 😛  (dev)
👉  rollup -c rollup.config.umd.js
Error loading /Users/montogeek/foo/bar/src/services/creditcard.js: /Users/montogeek/foo/bar/src/services/creditcard.js: Unexpected token (11:2)
SyntaxError: /Users/montogeek/foo/bar/src/services/creditcard.js: Unexpected token (11:2)
   9 |   CREDIT_CARD_EXPIRY_INVALID,
  10 |   CREDIT_CARD_CVC_INVALID
> 11 | } '../errorCodes';
     |   ^
  12 | import { config } from '../config';
  13 |
  14 | // API Endpoint
    at Parser.pp.raise (/Users/montogeek/foo/bar/node_modules/babylon/lib/parser/location.js:22:13)
    at Parser.pp.unexpected (/Users/montogeek/foo/bar/node_modules/babylon/lib/parser/util.js:89:8)
    at Parser.pp.expectContextual (/Users/montogeek/foo/bar/node_modules/babylon/lib/parser/util.js:57:39)
    at Parser.pp.parseImport (/Users/montogeek/foo/bar/node_modules/babylon/lib/parser/statement.js:1004:10)
    at Parser.pp.parseStatement (/Users/montogeek/foo/bar/node_modules/babylon/lib/parser/statement.js:141:56)
    at Parser.parseStatement (/Users/montogeek/foo/bar/node_modules/babylon/lib/plugins/flow.js:30:22)
    at Parser.pp.parseBlockBody (/Users/montogeek/foo/bar/node_modules/babylon/lib/parser/statement.js:529:21)
    at Parser.pp.parseTopLevel (/Users/montogeek/foo/bar/node_modules/babylon/lib/parser/statement.js:36:8)
    at Parser.parse (/Users/montogeek/foo/bar/node_modules/babylon/lib/parser/index.js:129:19)
    at parse (/Users/montogeek/foo/bar/node_modules/babylon/lib/index.js:47:47)
Type rollup --help for help, or visit https://github.com/rollup/rollup/wiki
```

File config:
rollup.config.js
```js
import babel from 'rollup-plugin-babel';
import nodeResolve from 'rollup-plugin-node-resolve';
import commonjs from 'rollup-plugin-commonjs';

const config = {
  entry: 'src/index.js',
  plugins: [
    babel({
      babelrc: false,
      presets: ['es2015-rollup'],
      plugins: ['transform-flow-strip-types', 'transform-object-rest-spread', 'transform-class-properties'],
      exclude: 'node_modules/**'
    }),
    commonjs({
      include: 'node_modules/**',
      namedExports: {
        'node_modules/moment/moment.js': ['moment']
      }
    }),
    nodeResolve({
      jsnext: true,
      browser: true,
      preferBuiltins: false
    })
  ]
}

export default config;
```

rollup.config.umd.js
```js
import config from './rollup.config';

config.format = 'umd';
config.dest = 'dist/project.umd.js';
config.moduleName = 'foo';
config.sourceMap = true;

export default config;
```

**Comment by Rich-Harris** [comment]
Thanks – have released `rollup-watch@2.2.0` with the first half of a fix for this – it will now emit an error event and recover. Now just need prettier logging

**Comment by montogeek** [comment]
Thanks @Rich-Harris
